net-im/spectrum2/spectrum2-2.0.10.1-r2.ebuild

```diff
@@ -18,6 +19,7 @@ IUSE="doc frotz irc mysql postgres purple sms +sqlite test twitter whatsapp xmpp
 REQUIRED_USE="|| ( mysql postgres sqlite )"
 
 RDEPEND="
+	test? ( ${PYTHON_DEPS} )
```

Comment:
test deps are in `RDEPEND` all of a sudden?

Reply:
Well, CI warnings says, I have to set PYTHON_DEPS in RDEPEND? Is this wrong in this case?

Reply:
If python is only used during build-time, then this ebuild should use `python-any-r1` instead. But if this ebuild is installing python files used at runtime, then I doubt that it is correct as conditional on `USE=test`?

Reply:
I see, than I used it indeed wrong. I changed not to ```python-any-r1``` and moved PYTHON_DEPS back to ```DEPEND```. Python is only used for tests. Is this correct this way?

Reply:
Yes but python-any-r1 doesn't have PYTHON_REQUIRED_USE.

Reply:
Thanks, removed.